crates/ty_python_semantic/resources/mdtest/suppressions/ty_ignore.md

```diff
@@ -127,7 +130,7 @@ a = 10 / 0  # ty: ignore[*-*]
 <!-- blacken-docs:off -->
 
 ```py
-a = 10 / 0  # ty: ignore[division-by-zero]       
+a = 10 / 0  # ty: ignore[division-by-zero]
             #                               ^^^^^^ trailing whitespace
```

Comment:
the comment here suggests to me that the trailing whitespace might have been intentional :P